src/hulls/graham.jl

```diff
@@ -21,7 +21,8 @@ struct GrahamScan <: HullMethod end
 function hull(points, ::GrahamScan)
   pₒ = first(points)
   Dim = embeddim(pₒ)
-  T = coordtype(pₒ)
+  ℒ = lentype(pₒ)
+  T = numtype(ℒ)
```

Comment:
it seems that T is unused now, please erase the line if that is the case

Reply:
It is being used on line 58:
```julia
while ∠(r[end - 1], r[end], B) > atol(T) && length(r) ≥ 3
```